scripts/run.sh

```diff
@@ -130,7 +130,8 @@ function dockerComposePull() {
 }
 
 function dockerPrune() {
-    docker image prune -f
+    docker image prune -f --filter="label=com.bitwarden.product=bitwarden"
+    # Perhaps we could add "-a", to recover disk space after automatic update ?
```

Comment:
no need for comments here, this could be part of the discussion in this PR comments